compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -1146,7 +1145,11 @@ TR::Register *OMR::X86::TreeEvaluator::fpConvertToLong(TR::Node *node, TR::Symbo
    TR::Node *child = node->getFirstChild();
 
    // TODO: Perform f2l using SSE.
+#ifdef J9_PROJECT_SPECIFIC
    if (child->getOpCode().isDouble() && cg->useSSEForDoublePrecision())
+#else
+   if (false)
+#endif
```

Comment:
Could you explain this sequence of code and why OpenJ9 and OMR need to go down different paths now?

Reply:
The SSE path uses a helper call that is not implemented by OMR. It is indeed a OpenJ9 specific code although it is not marked as `J9_PROJECT_SPECIFIC`.
We should definitely make the SSE path not relying on any J9 specific functions; however, I personally don't think we should hold enabling SSE/SSE2 solely because the path uses J9 specific code.